oda_api/api.py

```diff
@@ -866,7 +866,7 @@ def set_api_code(query_dict, url="www.astro.unige.ch/cdci/astrooda/dispatch-data
 
         _cmd_ = '%s\n' % _header
         _cmd_ += 'par_dict='
-        _cmd_ += '%s' % _api_dict
+        _cmd_ += '%s' % json.dumps(_api_dict, indent=0)
```

Comment:
why not put a conventional python indent? E.g. 4. And also the rest of the code could have formatting as conventional, spaces and all.